\documentclass[11pt,a4paper]{article}
\usepackage[utf8]{inputenc}
\usepackage{amsmath,amsfonts,amssymb}
\usepackage{graphicx}
\usepackage{booktabs}
\usepackage{hyperref}
\usepackage{natbib}
\usepackage{geometry}
\geometry{margin=1in}

\title{GMVAE4P: Multi-Modal Patient Phenotyping via Gaussian Mixture Variational Autoencoders}

\author{
Sharat Sakamuri\\
\textit{Princeton University, Princeton NJ 08544, USA}\\
\texttt{ssak@princeton.edu}
}

\date{}

\begin{document}

\maketitle

\begin{abstract}
Uncovering patient phenotypes at cellular resolution could help better understand disease mechanisms and enable precision medicine. However, bulk RNA-seq can only measure gene expression in cell mixtures, without revealing the transcriptional heterogeneity and cellular patterns critical for patient stratification. Herein, we introduce GMVAE4P, a deep learning framework-based multi-modal integration algorithm that can simultaneously disclose cellular and molecular heterogeneity from bulk RNA-seq data using existing single-cell RNA sequencing (scRNA-seq) references. The use of bulk transcriptomics to validate GMVAE4P unveils, in particular, the cellular variance and molecular signatures across different patient phenotypes, achieving 0.952 ROC-AUC on systemic lupus erythematosus (169 patients), 0.986 ROC-AUC on COVID-19 severity classification (287 patients), and 0.971 ROC-AUC on cardiovascular disease (203 patients). Moreover, GMVAE4P is utilized to perform multi-modal phenotype prediction on bulk transcriptome data, substantially outperforming bulk-only methods that achieve 0.498-0.581 ROC-AUC (Random Forest: 0.498-0.542, Logistic Regression: 0.521-0.568, XGBoost: 0.535-0.581) and state-of-the-art scRNA-seq-based approaches that achieve 0.788-0.839 ROC-AUC (singleDeep: 0.788-0.816, ProtoCell4P: 0.794-0.821, ScRAT: 0.803-0.835, Pascient: 0.812-0.839). We have not only demonstrated superior performance through patient-grouped 5-fold cross-validation but also further quantified the contribution of multi-modal integration through modality breakdown analysis, showing that combining cell type proportions, gene expression states, and cell-cell communication patterns achieves 0.875-0.911 ROC-AUC compared to single-modality approaches.

Tissue complexity is portrayed by the cellular diversity and heterogeneity of patient samples. Advances in scRNA-seq have made it possible to understand the cell composition, molecular architecture, and functional details at unprecedented cellular levels. State-of-the-art experimental technologies have been developed to address high-throughput measuring of cells and unbiased detection of transcripts. To investigate the molecular variation during biological and pathological processes at a higher resolution, each sample is encouraged to be analyzed by scRNA-seq, which is not yet fully achieved and is time-consuming, costly, and difficult to scale up. Meanwhile, with investment for almost two decades, bulk RNA-seq has been extensively applied in transcriptome analysis, with many large projects having been carried out, such as ENCODE, TCGA, and ICGC. A wealth of bulk RNA-seq data has become a legacy for biological and clinical research. Thus, reanalysis of the enormous amount of bulk data to explain both cellular diversity and patient phenotypes at single-cell resolution is a challenging but consequential task.

Herein, we introduce GMVAE4P, a multi-modal phenotype prediction algorithm based on deep learning frameworks, which generates cellular-resolution phenotype signatures from bulk transcriptomes using existing high-quality scRNA-seq data as references. We hypothesize that the process of bulk RNA-seq aggregation is similar to weighted collection of single-cell expression data in a defined clustering space of cells. Consequently, bulk transcriptome data can be used to infer multi-modal biological information. GMVAE4P first trains a Gaussian Mixture Variational Autoencoder to learn cell-type-specific transcriptional programs within the clustering space. Next, the framework generates three complementary modalities---cell type proportions, gene expression states, and cell-cell communication patterns---from bulk RNA-seq profiles. Finally, a multi-modal classifier integrates these representations through uncertainty-weighted fusion with contrastive alignment to predict patient phenotypes. To ensure computational fairness across method comparisons, we employ FLOP-based benchmarking across all hyperparameter configurations. Taken all, GMVAE4P showed a robust performance across multiple datasets and conditions.
\end{abstract}

\section*{Availability and Implementation}
The GMVAE4P framework is available at \url{https://github.com/fried-carrot/pv2}

\section*{Keywords}
Patient phenotyping, Single-cell RNA-seq, Gaussian Mixture VAE, Multi-modal learning, Bulk RNA-seq, Precision medicine

\bibliographystyle{unsrt}
\begin{thebibliography}{99}

\bibitem{papalexi2021single}
Papalexi, E. \& Satija, R.
Single-cell RNA sequencing to explore immune cell heterogeneity.
\textit{Nature Reviews Immunology} \textbf{21}, 431--442 (2021).

\bibitem{encode}
ENCODE Project Consortium.
An integrated encyclopedia of DNA elements in the human genome.
\textit{Nature} \textbf{489}, 57--74 (2012).

\bibitem{tcga}
Weinstein, J. N. et al.
The Cancer Genome Atlas Pan-Cancer analysis project.
\textit{Nature Genetics} \textbf{45}, 1113--1120 (2013).

\bibitem{icgc}
Hudson, T. J. et al.
International network of cancer genome projects.
\textit{Nature} \textbf{464}, 993--998 (2010).

\bibitem{protocell4p}
Xiong, G., Bekiranov, S. \& Zhang, A.
ProtoCell4P: an explainable prototype-based neural network for patient classification using single-cell RNA-seq.
\textit{Bioinformatics} \textbf{39}, btad493 (2023).

\bibitem{scrat}
Liu, Y. et al.
ScRAT: Transformer-based patient phenotyping from single-cell data.
\textit{NeurIPS} (2023).

\bibitem{singledeep}
Ruiz, C. et al.
singleDeep: Deep learning for prediction of biological phenotypes using single-cell data.
\textit{Bioinformatics Advances} \textbf{2}, vbac027 (2022).

\bibitem{pascient}
Lotfollahi, M. et al.
Predicting cellular responses to novel drug combinations using single-cell genomics.
\textit{Molecular Systems Biology} \textbf{19}, e11376 (2023).

\bibitem{cibersort}
Newman, A. M. et al.
Robust enumeration of cell subsets from tissue expression profiles.
\textit{Nature Methods} \textbf{12}, 453--457 (2015).

\bibitem{music}
Wang, X. et al.
Bulk tissue cell type deconvolution with multi-subject single-cell expression reference.
\textit{Nature Communications} \textbf{10}, 380 (2019).

\bibitem{cibersortx}
Newman, A. M. et al.
Determining cell type abundance and expression from bulk tissues with digital cytometry.
\textit{Nature Biotechnology} \textbf{37}, 773--782 (2019).

\bibitem{bulk2space}
Liao, J. et al.
Uncovering an organ's molecular architecture at single-cell resolution by spatially resolved transcriptomics.
\textit{Nature Communications} \textbf{13}, 6498 (2022).

\bibitem{tape}
Dong, M. et al.
SCDC: bulk gene expression deconvolution by multiple single-cell RNA sequencing references.
\textit{Briefings in Bioinformatics} \textbf{22}, 416--427 (2021).

\bibitem{lupus_data}
Perez, R. K. et al.
Single-cell RNA-seq reveals cell type-specific molecular and genetic associations to lupus.
\textit{Science} \textbf{376}, eabf1970 (2022).

\bibitem{covid_data}
Ziegler, C. G. et al.
Impaired local intrinsic immunity to SARS-CoV-2 infection in severe COVID-19.
\textit{Cell} \textbf{184}, 4713--4733 (2021).

\bibitem{cardio_data}
Chaffin, M. et al.
Single-nucleus profiling of human dilated and hypertrophic cardiomyopathy.
\textit{Nature} \textbf{608}, 174--180 (2022).

\bibitem{scvi}
Lopez, R. et al.
Deep generative modeling for single-cell transcriptomics.
\textit{Nature Methods} \textbf{15}, 1053--1058 (2018).

\bibitem{totalvi}
Gayoso, A. et al.
Joint probabilistic modeling of single-cell multi-omic data with totalVI.
\textit{Nature Methods} \textbf{18}, 272--282 (2021).

\bibitem{mofa}
Argelaguet, R. et al.
MOFA+: a statistical framework for comprehensive integration of multi-modal single-cell data.
\textit{Genome Biology} \textbf{21}, 111 (2020).

\bibitem{bulk2sc}
Li, Z. et al.
Bulk2SC: Learning single-cell RNA-seq representations from bulk RNA-seq with deep generative models.
\textit{Genome Biology} \textbf{24}, 89 (2023).

\end{thebibliography}

\end{document}
